inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/enums/KafkaScanStartupMode.java

```diff
@@ -25,7 +25,8 @@
  */
 public enum KafkaScanStartupMode {
     EARLIEST_OFFSET("earliest-offset"),
-    LATEST_OFFSET("latest-offset");
+    LATEST_OFFSET("latest-offset"),
+    SPECIFIC_OFFSET("specific-offsets");
```

Comment:
Why does this use `offsets` not `offset`?

Reply:
it is controlled by the sort connector.

Reply:
The API provided by the Flink community connector looks like this